druid/examples/widget_gallery.rs

```diff
@@ -23,7 +23,7 @@ use druid::{
     AppLauncher, Color, Data, Lens, Rect, Widget, WidgetExt, WidgetPod, WindowDesc,
 };
 
-#[cfg(not(target_arch = "wasm32"))]
+#[cfg(feature = "svg")]
```

Comment:
This seems reasonable, but unrelated?

Reply:
I think it was angering IntelliJ
Can split out